dev-docs/source/Architecture.rst

```diff
@@ -0,0 +1,180 @@
+============
+Architecture
+============
+
+.. contents::
+   :local:
+
+Summary
+-------
+
+Mantid is used for reduction and analysis of neutron and muon data.
```

Comment:
I'm in favour of not forgetting the data visualization aspect, I think it's important to put it in the opening sentence.

Reply:
Good point. We spend a lot of time writing those!